Tests/AppTests/__Snapshots__/AnalyzerTests/test_analyze.1.txt

```diff
@@ -1,97 +1,33 @@
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%H-%ct" "2.0.0"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%H-%ct\" \"2.0.0\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%H-%ct" "2.1.0"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%H-%ct\" \"2.1.0\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%H-%ct" "main"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%H-%ct\" \"main\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git log -n1 --format=format:"%ct"' at path: '.../github.com-foo-2'
-    - command: "git log -n1 --format=format:\"%ct\""
-    - path: ".../github.com-foo-2"
-  ▿ 'git reset "origin/main" --hard' at path: '.../github.com-foo-2'
-    - command: "git reset \"origin/main\" --hard"
-    - path: ".../github.com-foo-2"
-  ▿ 'git reset --hard' at path: '.../github.com-foo-2'
-    - command: "git reset --hard"
-    - path: ".../github.com-foo-2"
-  ▿ 'git rev-list --count HEAD' at path: '.../github.com-foo-2'
-    - command: "git rev-list --count HEAD"
-    - path: ".../github.com-foo-2"
-  ▿ 'git tag' at path: '.../github.com-foo-2'
-    - command: "git tag"
-    - path: ".../github.com-foo-2"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-2'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-2"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-2'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-2"
-  ▿ 'swift package dump-package' at path: '.../github.com-foo-2'
-    - command: "swift package dump-package"
-    - path: ".../github.com-foo-2"
+  - ".: clone https://github.com/foo/1"
+  - "./github.com-foo-2: reset"
+  - "./github.com-foo-2: clean"
+  - "./github.com-foo-2: fetch"
+  - "./github.com-foo-2: checkout"
+  - "./github.com-foo-2: reset to origin/main"
+  - "./github.com-foo-1: commitCount"
+  - "./github.com-foo-1: firstCommitDate"
+  - "./github.com-foo-1: lastCommitDate"
+  - "./github.com-foo-2: commitCount"
+  - "./github.com-foo-2: firstCommitDate"
+  - "./github.com-foo-2: lastCommitDate"
+  - "./github.com-foo-1: getTags"
+  - "./github.com-foo-2: getTags"
+  - "./github.com-foo-1: revisionInfo"
+  - "./github.com-foo-1: revisionInfo"
+  - "./github.com-foo-1: revisionInfo"
+  - "./github.com-foo-2: revisionInfo"
+  - "./github.com-foo-2: revisionInfo"
+  - "./github.com-foo-2: revisionInfo"
+  - "./github.com-foo-1: checkout"
+  - "./github.com-foo-1: dumpPackage"
+  - "./github.com-foo-1: checkout"
+  - "./github.com-foo-1: dumpPackage"
+  - "./github.com-foo-1: checkout"
+  - "./github.com-foo-1: dumpPackage"
+  - "./github.com-foo-2: checkout"
+  - "./github.com-foo-2: dumpPackage"
+  - "./github.com-foo-2: checkout"
+  - "./github.com-foo-2: dumpPackage"
+  - "./github.com-foo-2: checkout"
+  - "./github.com-foo-2: dumpPackage"
```

Comment:
Whether we decide to use `Command2` or not, one thing I will do (plus a couple of other things to make it stable), is filter the commands by package and create two separate snapshots. That should take care of the flickering when the commands are interleaved between the two packages.